src/Core/Domain/Order/Exception/OrderConstraintException.php

```diff
@@ -39,5 +39,5 @@ class OrderConstraintException extends OrderException
     /**
      * @var int Is used when invalid (not string) internal note provided
      */
-    public const INVALID_INTERNAL_NOTE = 2;
+    public const INVALID_PAYMENT_METHOD = 2;
```

Comment:
Nope, both are needed! I think `INVALID_PAYMENT_METHOD` was added in 177 recently but `INVALID_INTERNAL_NOTE` has been in 178 for a while

Reply:
:cry: